### PR TITLE
Add a simple tutorial of Bevy

### PR DESCRIPTION
**Pull request by fogarecious**
This pull request adds a simple tutorial of Bevy.

### CODE REVIEW

File: Learning/Simple_Tutorial_Of_Bevy.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name = "Simple Tutorial Of Bevy"
```
Comment (alice-i-cecile) [inline Learning/Simple_Tutorial_Of_Bevy.toml:1]:
```suggestion
name = "Simple Bevy Tutorial Bevy"
```

File: Learning/Simple_Tutorial_Of_Bevy.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+description = "A simple tutorial of the Bevy engine"
```
Comment (alice-i-cecile) [inline Learning/Simple_Tutorial_Of_Bevy.toml:2]:
```suggestion
description = "A simple tutorial for the Bevy engine"
```

**Review by alice-i-cecile — COMMENTED** [review]
Just some English grammar advice, feel free to accept or ignore :)

**Comment by fogarecious** [comment]
Have applied the suggestions. Thank you for the corrections.

**Pull request merged**